app/services/authorizer_cache.rb

```diff
@@ -1,14 +1,13 @@
 module AuthorizerCache
   def initialize_cache
-    @cache = HashWithIndifferentAccess.new do |h, k|
-      h[k] = HashWithIndifferentAccess.new
+    @cache = Hash.new do |h, k|
+      h[k] = {}
     end
   end
 
   def collection_cache_lookup(subject, permission)
-    collection = @cache[subject.class.to_s][permission] ||=
+    collection = @cache[subject.class.to_s][permission.to_s] ||=
```

Comment:
I was thinking the same, indifferent access is often used for no reason. It could be even flattened to avoid creating many Hash objects, however, probably not worth it.